src/ReactWrapper.jsx

```diff
@@ -175,6 +177,8 @@ class ReactWrapper {
     if (this.root !== this) {
       throw new Error('ReactWrapper::instance() can only be called on the root');
     }
+    if (!this.component.state.mount) return instanceMap.get(this.component);
```

Comment:
Doing so seems to cause it to fail in React 13. I'm not sure exactly why, though. Might have something to do with a change in the way empty components were handled?